hooks/useOracleSynced/useOracleSynced.ts

```diff
@@ -9,27 +8,15 @@ export function useOracleSynced(
   oracleKind: OraclePriceType,
 ) {
   const oracleInfo = useOracleInfo(oracleKind);
-  const { jsonRpcProvider } = useConfig();
 
   const [synced, setSynced] = useState<boolean>(false);
-  const [blockTimestamp, setBlockTimestamp] = useState<number>(0);
-
-  useEffect(() => {
-    const fetchBlockTimestamp = async () => {
```

Comment:
we were previously making an extra rpc call per vault for this 😬 using the timestamp context/hook now to save that call